bin/node/src/service.rs

```diff
@@ -399,6 +399,8 @@ pub fn new_authority(
             .unwrap_or(usize::MAX),
     };
 
+    let validator_address_cache = None;
```

Comment:
Hmm, why is this `Option`al?

Reply:
Most users won't be interested in getting these network details, and we will need it only for rpc nodes. The decision if we create a `Noop` updater or the real one must be done in `bin/`, but the updater should be created in `finality-aleph`, so I've chosen to make `validator_address_cache` (which also must be created in `bin/`, as it will be used in rpc) optional and select the appropriate variant of updater based on that.